WooCommerce/Classes/ViewRelated/ReusableViews/EnhancedTextView.swift

```diff
@@ -57,14 +57,24 @@ final class EnhancedTextView: UITextView {
 //
 private extension EnhancedTextView {
     func configurePlaceholderLabel() {
-        let frameRect = CGRect(x: Constants.margin, y: Constants.margin, width: bounds.width, height: bounds.height)
-        placeholderLabel = UILabel(frame: frameRect)
-        if let unwrappedLabel = placeholderLabel {
-            addSubview(unwrappedLabel)
-        }
-        placeholderLabel?.numberOfLines = 0
-        placeholderLabel?.applyBodyStyle()
-        placeholderLabel?.textColor = .textSubtle
+        placeholderLabel = {
+            let label = UILabel(frame: bounds)
+            label.translatesAutoresizingMaskIntoConstraints = false
+            addSubview(label)
+
+            // Make placeholder left/right margins same as for the text container
+            NSLayoutConstraint.activate([
+                label.leadingAnchor.constraint(equalTo: leadingAnchor, constant: textContainer.lineFragmentPadding + textContainerInset.left),
+                label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -(textContainer.lineFragmentPadding + textContainerInset.right)),
+                label.topAnchor.constraint(equalTo: topAnchor, constant: Constants.margin)
```

Comment:
If the text view height is bigger than one line and placeholder label is pinned to all edges, placeholder label would vertically centre the text and we want it to be top aligned.
I updated constraints to use width / heigh of the text view in https://github.com/woocommerce/woocommerce-ios/pull/3407/commits/8d021637ff4dff2e6a92e224f1ce0826404e9840 as I figured that using trailing constraint doesn't work well because text view is basically a scroll view.